app/helpers/exception_hunter/errors_helper.rb

```diff
@@ -0,0 +1,4 @@
+module ExceptionHunter
+  module ErrorsHelper
```

Comment:
🤔 what's this for?

Reply:
Helpers get included in every view. They provide helper methods to format strings and the likes 

https://www.rubyguides.com/2020/01/rails-helpers/